mlflow/models/utils.py

```diff
@@ -172,25 +189,40 @@ def _handle_dataframe_input(df):
             return result
 
         example_filename = "input_example.json"
+        params_filename = "example_params.json"
```

Comment:
```suggestion
        params_filename = "params_example.json"
```

wondering which one makes more sense, `example_params` vs. `params_example`.

Reply:
😆 I think `params_example`